src/commands/edge_app.rs

```diff
@@ -1242,11 +1242,11 @@ mod tests {
         let command = EdgeAppCommand::new(authentication);
         let manifest = EdgeAppManifest {
             app_id: "01H2QZ6Z8WXWNDC0KQ198XCZEW".to_string(),
-            user_version: "1".to_string(),
-            description: "asdf".to_string(),
-            icon: "asdf".to_string(),
-            author: "asdf".to_string(),
-            homepage_url: "asdfasdf".to_string(),
+            user_version: Some("1".to_string()),
```

Comment:
This is not related to this task 100%. But can you move creation of EDgeAppManifest to a separtate function for those that use the same values please?